### DISCUSSION TITLE
Add Backstage social icon

### DISCUSSION THREAD

**Discussion by jdevega** [withastro/starlight#2597] [Feature Requests]
### What version of `starlight` are you using?

0.26.1

### What is your idea?

Add Backstage ( https://backstage.io/ ) social icon to the list.

### Why is this feature necessary?

I am going to switch from Stack Overflow to Backstage and maybe others are going to use it too.

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by HiDeoo** [comment]
Thanks for the feedback.

It looks like this request did not get a lot of traction. We are currently working on adding support for custom icons in a future release which would solve this use case. In the meantime, you can follow this [guide](https://hideoo.dev/notes/starlight-custom-social-links) which explains how to add social links with custom icons to Starlight.
